x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -125,133 +125,6 @@
     "advancedSettings.searchBar.unableToParseQueryErrorMessage": "クエリをパースできません",
     "advancedSettings.searchBarAriaLabel": "高度な設定を検索",
     "advancedSettings.voiceAnnouncement.ariaLabel": "詳細設定結果情報",
-    "apmOss.tutorial.apmAgents.statusCheck.btnLabel": "エージェントステータスを確認",
```

Comment:
@Bamieh note that these are all removed from `apmOss` but the same translations exist now under `xpack.apm`, so they shouldn't need to be re-translated from scratch.